examples/pytorch/translation/run_translation.py

```diff
@@ -378,7 +378,7 @@ def main():
             token=model_args.token,
         )
     # See more about loading any type of standard or custom dataset (from files, python dict, pandas DataFrame, etc) at
-    # https://huggingface.co/docs/datasets/v2.13.1/loading.
+    # https://huggingface.co/docs/datasets/loading.
```

Comment:
Remember for future
https://github.com/huggingface/transformers/pull/24594#discussion_r1248095853